### ISSUE TITLE
XAML Standard API Specificity

### ISSUE THREAD

**Issue by jkoritzinsky** [Microsoft/xaml-standard#227]
As XAML Standard is developed, can the standard spec file include the specific list of members that each control must support to implement the XAML Standard?

As a developer of another XAML-based UI that's planning on trying to support XAML Standard, it's really difficult to make sure we're implementing our XAML Standard controls with the correct behavior without knowing exactly which properties/events are required by the standard.

**Comment by insinfo** [comment]
@jkoritzinsky I agree and support +1

**Comment by touseefbsb** [comment]
yes set of events, properties and methods should be standardized for sure

**Comment by 4creators** [comment]
That's an obvious requirement for any reasonable interoperability via standard specification which even indirectly involves an API.
